tests/apollo/util/skvbc.py

```diff
@@ -262,6 +289,41 @@ async def fill_and_wait_for_checkpoint(
                 expected_checkpoint_num=lambda ecn: ecn == checkpoint_before + num_of_checkpoints_to_add,
                 verify_checkpoint_persistency=verify_checkpoint_persistency,
                 assert_state_transfer_not_started=assert_state_transfer_not_started)
+    
+    async def run_concurrent_txns_and_wait_for_checkpoint(
+            self, initial_nodes,
```

Comment:
nit: The name initial_nodes is not a good one in my opinion, as this is a service function, and know nothing about the fact you are doing state transfer. Maybe it needs to wait for all replicas for example? Why not just call it replicas?